src/main/java/com/salesforce/mirus/metrics/MirrorJmxReporter.java

```diff
@@ -127,7 +173,24 @@ private Sensor createTopicSensor(String topic) {
     return sensor;
   }
 
+  private Sensor createHistogramSensor(String topic, String bucket) {
+    Map<String, String> tags = new LinkedHashMap<>();
+    tags.put("topic", topic);
+    tags.put("bucket", bucket);
+
+    Sensor sensor = metrics.sensor(histogramLatencySensorName(topic, bucket));
+    sensor.add(
+        metrics.metricInstance(HISTOGRAM_LATENCY, tags),
+        new Rate(TimeUnit.SECONDS, new WindowedSum()));
+
+    return sensor;
+  }
+
   private String replicationLatencySensorName(String topic) {
     return topic + "-" + "replication-latency";
   }
+
+  private String histogramLatencySensorName(String topic, String bucket) {
+    return topic + "-" + bucket + "-" + "histogram-latency";
```

Comment:
I find the order strange here, we generally go from less to more specific in naming. How about `topic + "-" + "histogram-latency" + "-" + bucket `?

Reply:
I think this name is align with the sensor pattern in S3 codebase which has topic and connector.

Reply:
@YongGang OK, I guess the sensor name doesn't matter too much anyway - the sensor names and tags look good.